tests/browser/test_emscripten_unwind_to_js_event_loop.c

```diff
@@ -1,22 +1,17 @@
 #include <emscripten/html5.h>
 #include <stdio.h>
+#include <stdlib.h>
 
-void timeout(void *userData)
-{
-	printf("Got timeout handler\n");
-#ifdef REPORT_RESULT
-	// Test passed
-	REPORT_RESULT(1);
-#endif
+void timeout(void *userData) {
+  printf("Got timeout handler\n");
+  // Test passed
+  exit(0);
 }
 
-int main()
-{
-	emscripten_set_timeout(timeout, 2000, 0);
-	emscripten_unwind_to_js_event_loop();
-	printf("This should not be called!\n");
-#ifdef REPORT_RESULT
-	// Should not reach here
-	REPORT_RESULT(-1);
-#endif
+int main() {
+  emscripten_set_timeout(timeout, 2000, 0);
+  emscripten_unwind_to_js_event_loop();
+  printf("This should not be called!\n");
+  // Should not reach here
+  return 99;
```

Comment:
Do all these `return 99`s mean this cannot be reached? Would it be better to make `emscripten_unreachable()` or something in the test framework?